app/views/users/new.html.haml

```diff
@@ -1 +1,14 @@
-%h1 hello 2 
\ No newline at end of file
+%form
+	%header
+		=image_tag "micolet-logo.svg"
+		%p Almost new fashion with guarantee
+	.title
+		%p Welcome! Soon we will launch our site
+		%p in this country.
+		%p.subs Suscribe to receive our newsletter.
+	
+	.input_section
```

Comment:
Instead of a `div` this could be a main tag.

Reply:
change it to a  `form`